src/PSR-12/CodeStyle.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * File doc
+ * PHP version 7.4
+ *
+ * @category Category
+ * @package  Package
+ * @author   Octavio <octavio.marchi@gmail.com>
+ * @license  http://www.opensource.org/licenses/mit-license.html  MIT License
+ * @link     5
+ */
+
+/**
+ * Class doc
+ *
+ * @category Category
+ * @package  Package
+ * @author   Octavio <octavio.marchi@gmail.com>
+ * @license  http://www.opensource.org/licenses/mit-license.html  MIT License
+ * @link     5
+ */
+class CodeStyle
+{
+    var $Foo = "bar";
+
+    /**
+     * small description
+     *
+     * @param $bar bar
+     *
+     * @return void
+     */
+    function foo($bar)
```

Comment:
:no_entry_sign: **Error**: Tabs must be used to indent lines; spaces are not allowed (*Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed*).